Allow plugins.sh to be stowed or tracked in dotfiles as symlinks

### op CLI version

2.19.0-beta.01

### Goal or desired behavior

Currently, the CLI expects `~/.config/op/plugins.sh` to be a file. I would like to keep it in my dotfiles and symlink it to the requisite location.



### Current behavior

`[ERROR] 2023/09/03 01:15:00 Can't continue. We can't safely access "/Users/mrugesh/.config/op/plugins.sh" because it's a symbolic link. Replace the linked file with the original file and try again.`

### Relevant log output

_No response_

--- a/README.md
+++ b/README.md
@@ -69,7 +69,7 @@ Still not sure where or how to begin? We're happy to help! You can:
 By default, `op plugin init` writes its configuration to `~/.config/op/plugins.sh`
 and manages that file for you. If you'd rather keep your plugin configuration in
 your own dotfiles (for example, tracked in version control), you can define the
-shell functions yourself instead.
+shell functions yourself and source them in your shell's configuration file. 
 
 ### Step 1: Define the plugin functions
 Shell functions for each shell should follow this pattern. In this case, 
@@ -92,18 +92,18 @@ end
 
 ### Step 2: Tell `op` your plugins are already configured
 
-Set `OP_PLUGINS_SOURCED=1` so 1Password CLI knows the shell plugins have
+Set `OP_PLUGIN_ALIASES_SOURCED=1` so 1Password CLI knows the shell plugins have
 already been set up:
 
 ### bash / zsh
 
 ```bash
-export OP_PLUGINS_SOURCED="1"
+export OP_PLUGIN_ALIASES_SOURCED="1"
 ```
 
 ### fish
 ```fish
-set -x OP_PLUGINS_SOURCED 1
+set -x OP_PLUGIN_ALIASES_SOURCED 1
 ```
 
 > **Note:** `op plugin init <plugin>` edits `~/.config/op/plugins.sh` directly and won't update your own dotfiles, so configure plugins manually as shown above.

--- a/README.md
+++ b/README.md
@@ -63,6 +63,52 @@ Still not sure where or how to begin? We're happy to help! You can:
 - Book a free [pairing session](https://calendly.com/d/grs-x2h-pmb/1password-shell-plugins-pairing-session) with one of our developers
 - Join the [Developer Slack workspace](https://developer.1password.com/joinslack), and ask us any questions there
 
+## Advanced Configuration
+### Managing shell plugins in your own dotfiles
+
+By default, `op plugin init` writes its configuration to `~/.config/op/plugins.sh`
+and manages that file for you. If you'd rather keep your plugin configuration in
+your own dotfiles (for example, tracked in version control), you can define the
+shell functions yourself instead.
+
+### 1. Define the plugin functions
+Shell functions for each shell should follow this pattern. In this case, 
+
+### bash / zsh
+
+```bash
+gh() {
+  op plugin run -- gh "$@"
+}
+```
+
+### fish
+
+```fish
+function gh --wraps gh --description "1Password shell plugin for GitHub CLI"
+  op plugin run -- gh $argv
+end
+```
+
+### 2. Tell `op` your plugins are already configured
+
+Set `OP_PLUGINS_SOURCED=1` so the 1Password CLI knows the shell plugins have
+already been set up:
+
+### bash / zsh
+
+```bash
+export OP_PLUGINS_SOURCED="1"
+```
+
+### fish
+```fish
+set -x OP_PLUGINS_SOURCED 1
+```
+
+> **Note:** `op plugin init <plugin>` edits `~/.config/op/plugins.sh` directly and won't update your own dotfiles, so configure plugins manually as shown above.
+
+
 ## 💙 Community & Support
 
 - File an [issue](https://github.com/1Password/shell-plugins/issues/new/choose) for bugs and feature requests

--- a/README.md
+++ b/README.md
@@ -71,7 +71,7 @@ and manages that file for you. If you'd rather keep your plugin configuration in
 your own dotfiles (for example, tracked in version control), you can define the
 shell functions yourself instead.
 
-### 1. Define the plugin functions
+### Step 1: Define the plugin functions
 Shell functions for each shell should follow this pattern. In this case, 
 
 ### bash / zsh
@@ -90,9 +90,9 @@ function gh --wraps gh --description "1Password shell plugin for GitHub CLI"
 end
 ```
 
-### 2. Tell `op` your plugins are already configured
+### Step 2: Tell `op` your plugins are already configured
 
-Set `OP_PLUGINS_SOURCED=1` so the 1Password CLI knows the shell plugins have
+Set `OP_PLUGINS_SOURCED=1` so 1Password CLI knows the shell plugins have
 already been set up:
 
 ### bash / zsh

